compel/arch/riscv64/src/lib/infect.c

```diff
@@ -181,20 +181,28 @@ int arch_fetch_sas(struct parasite_ctl *ctl, struct rt_sigframe *s)
  * Task size is the maximum virtual address space size that a process can occupy in the memory
  * Refer to linux kernel arch/riscv/include/asm/pgtable.h,
  * task size is:
- * -     0x9fc00000 (~2.5GB) for RV32.
- * -   0x4000000000 ( 256GB) for RV64 using SV39 mmu
- * - 0x800000000000 ( 128TB) for RV64 using SV48 mmu
+ * -        0x9fc00000	(~2.5GB) for RV32.
+ * -      0x4000000000	( 256GB) for RV64 using SV39 mmu
+ * -    0x800000000000	( 128TB) for RV64 using SV48 mmu
+ * - 0x100000000000000	(  64PB) for RV64 using SV57 mmu
  *
  * Note that PGDIR_SIZE must evenly divide TASK_SIZE since "RISC-V
  * Instruction Set Manual Volume II: Privileged Architecture" states that
  * "load and store effective addresses, which are 64bits, must have bits
  * 63–48 all equal to bit 47, or else a page-fault exception will occur."
```

Comment:
This comment was copied from kernel `arch/riscv/include/asm/pgtable.h`, where it explains the rationale behind `#define TASK_SIZE_64 (PGDIR_SIZE * PTRS_PER_PGD / 2)`. I think we can either remove or replace this comment as it looks out of place in our context :)

Reply:
Hi Haorong,

thanks, fixed!